core/file_api.php

```diff
@@ -69,8 +69,47 @@ function file_attach_files( $p_bug_id, $p_files, $p_bugnote_id = 0 ) {
 	}
 
 	$t_file_infos = array();
+	$t_zip_attachments = config_get( 'zip_attachments' );
+	$t_zip_go = false;
+	if ( ON == $t_zip_attachments && extension_loaded( 'zip' ) ) {
+		$t_excluded_files = array_map( 'strtolower', config_get( 'zip_excludes' ) );
+		$t_zip_go = true;
+	}
 	foreach( $p_files as $t_file ) {
 		if( !empty( $t_file['name'] ) ) {
+			if ( $t_file['size'] > config_get( 'zip_threshold' ) ) {
```

Comment:
Move config_get() out of loop, initialize variable if $t_zip_attachments == true

Reply:
Adjusted accordingly